README.md

```diff
@@ -29,6 +29,8 @@ appcenter_upload(
 )
 ```
 
+The action parameters `api_token` and `owner_name` can also be omitted when their values are [set as environment variables](https://docs.fastlane.tools/advanced/#environment-variables) `APPCENTER_API_TOKEN` and `APPCENTER_OWNER_NAME`.
```

Comment:
@janpio Why is this only documenting these two very specific environment variables? There are a bunch more and imo especially `APPCENTER_APP_NAME` would fit in great.

Reply:
I'd make it a list of environment variables. Any more suggestions @lumaxis ?

Reply:
A list sounds good 👍 Or at least a hint where to look up all available variables.

Reply:
These are the ones of the example you normally would like to set to get rid of these two parameters as they never change in a project. app_name (different apps for debug and release apps) and apk (different build...) will change all the time.